tests/ops/models/test_privacy_experience.py

```diff
@@ -110,6 +112,7 @@ def test_create_privacy_experience_config(self, db, privacy_notice):
         assert history.name == "My New Experience Config"
         assert history.origin is None
         assert history.privacy_preferences_link_label == "Manage preferences"
+        assert history.modal_link_label == "Manage my consent preferences"
```

Comment:
Thanks for adding these assertions on both the translation record and the historical record 👍